src/vsr/clock.zig

```diff
@@ -473,9 +478,11 @@ pub fn ClockType(comptime Time: type) type {
             // for the operator, as the counterpoint to `no agreement on cluster time`.
             if (self.epoch.synchronized == null and self.window.synchronized != null) {
                 const new_interval = self.window.synchronized.?;
+                const accuracy = new_interval.upper_bound - new_interval.lower_bound;
```

Comment:
Perhaps we could avoid introducing the non-big endian `new_interval` into the scope entirely, using `window.synchronized` inline instead.